oracle-plugin/widgets/Oracle-connector.json

```diff
@@ -59,6 +59,18 @@
           "label": "SID/Service Name/TNS Connect Descriptor",
           "description": "Oracle connection point (Database name, Service name, or TNS Connect Descriptor)",
           "name": "database"
+        },
+        {
+          "widget-type": "select",
+          "label": "Transaction Isolation Level",
```

Comment:
Similarly here, does it make sense to move the position of this widget below `Role` widget?

Reply:
fixed